Add composite key via charm-helpers

AdamIsrael opened the pull request.

AdamIsrael committed:
Add composite key via charm-helpers

marcoceppi commented on actions/stress2json.py:
```diff
@@ -89,15 +89,23 @@ def parse_stress_output():
+            'units',
+            'desc'
+        )
+    except:
```
We should make a note to drop this eventually? Since chlp benchmark/charmtools is no longer just being installed by collectd and instead being installed by the charm, this won't really be required going forward

AdamIsrael replied:
Correct, we're installing it via pip. I'll clean it up in a mo'.

marcoceppi commented on actions/stress2json.py:
```diff
@@ -89,15 +89,23 @@ def parse_stress_output():
+        from charmhelpers.contrib.benchmark import Benchmark
+        Benchmark.set_composite_score(
+            total['interval_op_rate-rate'],
+            'units',
```
Should this be `ops/second` ?

AdamIsrael committed:
Composite key fixes

Fix units, remove try block since set_composite_score is available in
the latest charmhelpers
